OPENCOLLECTIVE.rst

```diff
@@ -0,0 +1,43 @@
+==============
+OpenCollective
+==============
+
+pytest has a collective setup at `OpenCollective`_. This document describes how the core team manages
+OpenCollective-related activities.
+
+What is it
+==========
+
+Open Collective is an online funding platform for open and transparent communities.
+It provide tools to raise money and share your finances in full transparency.
+
+It is the platform of choice for individuals and companies that want to make one-time or
+monthly donations directly to the project.
+
+Funds
+=====
+
+The OpenCollective funds donated to pytest will be used fund overall maintenance and future
+gatherings of pytest developers (Sprints).
+
+`Core contributors`_ which are contributing on a continuous basis are free to submit invoices
```

Comment:
This point has not been discussed at all, but I feel this is a good start. Would love to hear other's opinions on this.

Reply:
i  believe that unless we notice abuse of this its fine, and right now that seems excessively unlikely to happen